README.md

```diff
@@ -1,3 +1,247 @@
-# swift-quickstart-platform
+# PubNub Swift Sample App
 
 This repository contains the files required to run the [PubNub Swift Quickstart](https://www.pubnub.com/docs/platform/quickstarts/swift).
```

Comment:
Do we still need this line?

Reply:
I say yes. You're almost certainly arriving here from the Quickstart page on the platform docs. This is a quiet reminder that you're arrived in the right spot. Probably also doesn't hurt for SEO purposes, either.